development/build/scripts.js

```diff
@@ -372,6 +373,7 @@ function createScriptTasks({ browserPlatforms, livereload }) {
         INFURA_PROJECT_ID: opts.testing
           ? '00000000000000000000000000000000'
           : conf.INFURA_PROJECT_ID,
+        SEGMENT_HOST: conf.SEGMENT_HOST ? conf.SEGMENT_HOST : undefined,
```

Comment:
Can we do `conf.SEGMENT_HOST ?? undefined`, or just `conf.SEGMENT_HOST`?

Reply:
I had tried to use `conf.SEGMENT_HOST ?? undefined` initially, but this script isn't run through `babel` so it failed (`??` isn't supported by Node v10).

I suppose just `conf.SEGMENT_HOST` would work as well :thinking: I wanted to ensure this was left `undefined` rather than as an empty string if it wasn't set. I'll check to see what the default is - I never tested it.

Reply:
It was unnecessary after all! It defaults to `undefined` if it's not set. I have removed this ternary in https://github.com/MetaMask/metamask-extension/pull/9783/commits/9d5a838abfdea196c8f73f1288031cf30518ce63